tests/test_global.py

```diff
@@ -0,0 +1,80 @@
+import glob
+import pickle
+import os
+import contextlib
+import io
+from satellite_images_nso_tif_model_iterator.tif_model_iterator import (
+    tif_model_iterator,
+)
+from satellite_images_nso_tif_model_iterator.filenames.file_name_generator import (
+    OutputFileNameGenerator,
+)
+import pandas as pd
+
+# All these variables are local!
+model_path = model_path = (
+    "C:/repos/satellite-images-nso-datascience/saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
+)
+
+
+test_tif_files_dir = "E:/output/test/Nieuwkoopse_plassen/*SV*.tif"
```

Comment:
I would also recommend making sure these files are added and in a relative path. (same for the 2 paths below)

Reply:
somehow setting files keeps not being read with pytest, for now hardcoded in a todo file